app/site/questions.js

```diff
@@ -6,7 +6,11 @@ import ResidentsWithDisabilities from '../components/questions/ResidentsWithDisa
 import BelongingRates from '../components/questions/BelongingRates'
 import RentBurdenedRates from '../components/questions/RentBurdenRates'
 import SidewalkCafePermitsYears from '../components/questions/SidewalkCafePermitsYears'
+<<<<<<< HEAD
+import DisabilitiesAndTransportation from '../components/questions/DisabilitiesAndTransportation'
+=======
```

Comment:
There are some merge markers in this file, make sure you resolve the merge conflicts!

Reply:
![image](https://user-images.githubusercontent.com/36111703/127757458-26773713-8b44-4a3a-bd8c-669cb0c913b7.png)